src/components/AddBooks/AddBooks.js

```diff
@@ -1,21 +1,41 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 } from 'uuid';
 import PropTypes from 'prop-types';
+import { addBook } from '../../redux/books/books';
 
 const AddBooks = (props) => {
+  const dispatch = useDispatch();
   const { categories } = props;
 
```

Comment:
Please add a local state for `title` and `author` to `AddBooks` and please use the `onChange` event for these attributes so that the state is updated every time their value is changed.

Reply:
Done.